### PR TITLE
[WIP] streamline the whole documentation

### PR DESCRIPTION
**Pull request by OskarStark**
Refs sonata-project/SonataAdminBundle#3310

What did I do?

1. removed to include the ```sonata_timeline.yml``` config file, because in the whole documentation we use the ```app/config/config.yml``` file
2. better structure with subheadlines
3. rewrote the easy extend bundle step like in the page bundle
4. typos

please review this PR!

Thank you :smile:

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
You have extra commits here.

**Comment by OskarStark** [comment]
extra commits fixed @Soullivaneuh

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Timeline Bundle
 ===============
 
-The ``SonataTimelineBundle`` integrates the ``SpyTimelineBundle`` into Sonata. Timeline allows to create an action wall
+The ``SonataTimelineBundle`` integrates the ``SpyTimelineBundle`` into ``SonataAdmin``. Timeline allows to create an action wall
 for a user or a group of user. The current implementation integrate the basic features of the ``SpyTimelineBundle``:
```
Comment (javiereguiluz) [inline Resources/doc/index.rst]: `group of user` -> ``group of users`

File: Resources/doc/reference/configuration.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                component:            '%spy_timeline.class.component%'
+                actionComponent:      ~
+                action_component:     '%spy_timeline.class.action_component%'
+                action:               '%spy_timeline.class.action%'
+                timeline:             '%spy_timeline.class.timeline%'
+
+Customize the Timeline Block
+----------------------------
+
+You can customize the ``title`` (default: ``Latest Actions``) of the block by using this config options:
```
Comment (javiereguiluz) [inline Resources/doc/reference/configuration.rst]: `this ... options` -> `these ... options`

File: Resources/doc/reference/installation.rst [hunk 3]
```diff
@@ -1,124 +1,152 @@
...
 
-    //composer.json
-    "require": {
-    //...
-        "sonata-project/timeline-bundle": "~2.2@dev",
-    //...
-    }
+    $ php composer.phar require sonata-project/timeline-bundle
+
+Alternatively, you could add a dependency into your `composer.json` file directly.
```
Comment (javiereguiluz) [inline Resources/doc/reference/installation.rst]:
```
`composer.json`
```

should be:

```
``composer.json``
```

File: Resources/doc/reference/installation.rst [hunk 4]
```diff
@@ -1,124 +1,152 @@
...
+
+.. note::
 
+    This will install the `SpyTimelineBundle <https://github.com/stephpy/timeline-bundle>`, too.
```
Comment (javiereguiluz) [inline Resources/doc/reference/installation.rst]: The link needs a trailing underscore: `xxx <yyy>`_

File: Resources/doc/reference/installation.rst (hunk 3 above)
Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: i have to lern the difference, for me it isn't clear yet when to use one or two ticks :smile:

**Comment by OskarStark** [comment]
thank you @javiereguiluz i made the changes

**Pull request merged**